Update list of signature algorithms

itsgreat2misha opened the pull request:
Builds on [jdobes' PR](https://github.com/runDMCA/home-assistant-mazda/pull/8) to fix #5 by updating the list of supported algorithms to reflect the supported list provided from the client hello:
<img width="329" alt="image" src="https://github.com/user-attachments/assets/fbe2f8ae-3af3-474d-b993-cce1768e628f" />

itsgreat2misha committed:
Update __init__.py

Fix for ha 2024.7

jdobes committed:
fix: add library for advanced manipulation with SSLContext

source: https://github.com/ramikg/ssl-context-configurator

jdobes committed:
fix: use fixed list of signature algorithms that are expected by Mazda servers

itsgreat2misha committed:
Merge pull request #1 from jdobes/signature_algorithms_fix

Signature algorithms fix

itsgreat2misha committed:
Update connection.py

test modify signature algorithms to reflect supported ones from packet capture client hello

StephenMilner commented:
@galperinm - The integrations seems to have stopped working with these algorithms. It's timeouts now.

Any chance you can have another go with Wireshark to see what's changed?

itsgreat2misha committed:
Update connection.py

Set cipher suites and signature algorithms to match what's captured from my app on Android
